editor/plugins/script_editor_plugin.cpp

```diff
@@ -2681,6 +2687,12 @@ ScriptEditor::ScriptEditor(EditorNode *p_editor) {
 	members_overview_buttons_hbox = memnew(HBoxContainer);
 	members_overview_vbox->add_child(members_overview_buttons_hbox);
 
+	filename = memnew(Label);
+	filename->set_clip_text(true);
+	filename->set_h_size_flags(SIZE_EXPAND_FILL);
+	filename->add_style_override("normal", EditorNode::get_singleton()->get_gui_base()->get_stylebox("normal", "LineEdit"));
```

Comment:
Isn't it confusing to have it styled as a `LineEdit`? My first impression on your GIF was that it would be a search bar.

Reply:
I followed it from filename field on import dock.
would you point me a better example?